hydrophone_demo.ipynb

```diff
@@ -0,0 +1,502 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<img src='imgs/ooipy_banner.png' align=\"center\">\n",
+    "\n",
+    "# OOIPY Demo\n",
+    "\n",
+    "This Jupyter Notebook walks through the basic functionality of OOIpy\n",
+    "\n",
+    "## In this demo you will learn:\n",
+    "- How to download broadband (Fs=64kHz) and low frequency (Fs = 200) Hz from OOI server\n",
+    "- How to compute spectrograms and PSDs from the data\n",
+    "- How to save spectrograms and PSDs to pickle files\n",
+    "- How to plot hydrophone data, spectrograms and PSDs\n",
+    "- How to save Hydrophone Data to .wav file"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Environment Setup"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import ooipy\n",
+    "import datetime\n",
+    "from matplotlib import pyplot as plt\n",
+    "\n",
+    "# Some Jupyter magic so that the notebook will reload external python modules;\n",
+    "# see http://stackoverflow.com/questions/1907993/autoreload-of-modules-in-ipython\n",
+    "%load_ext autoreload\n",
+    "%autoreload 2"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Download Data from OOI\n",
+    "\n",
+    "### Valid Hydrophone Nodes: \n",
+    "#### Broadband\n",
+    "- '\\LJ01D' - Oregon Shelf Base Seafloor\n",
```

Comment:
I think we should remove the "\" here (and in the next few lines) to not confuse people

Reply:
Also it seems that the PEP8 failed due to an install error. It seems to use python 3.7.10 but PEP8 only supports 3.7.11 or some other versions. Do you have any idea how that happened?

Reply:
ok I think I soled the python version issue in the other PR

Reply:
I updated the demo with the same info from readthedocs, so it should be clear now.